BankSDK/GiniBankSDKExample/GiniBankSDKExample/AppDelegate.swift

```diff
@@ -24,7 +26,18 @@ var window: UIWindow?
     func application(_ app: UIApplication,
                      open url: URL,
                      options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
-        coordinator.processExternalDocument(withUrl: url, sourceApplication: options[.sourceApplication] as? String)
+        // Coming from Photos extension app
+        if url.absoluteString == appSheme {
+            if let userDefaults = UserDefaults(suiteName: appGroupName) {
```

Comment:
[Suggestion] Personally I prefer one if with all the conditions since we are not doing anything special for any other case.